ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/manager/EthScheduler.java

```diff
@@ -62,12 +65,33 @@ public EthScheduler(
       final int txWorkerCount,
       final int computationWorkerCount,
       final MetricsSystem metricsSystem) {
+    this(
+        syncWorkerCount,
+        txWorkerCount,
+        BoundedTimedQueue.Default.CAPACITY,
```

Comment:
I'd make the capacity explicitly transaction specific - we should be selecting a custom queue size that suits the use case every time we create a bounded queue, not just using a general default.

Reply:
Done.